Document/0x04f-Testing-Network-Communication.md

```diff
@@ -407,7 +407,7 @@ The following configuration files need to be changed and adjusted accordingly:
 
 #### MITM Attack
 
-To be able to get a man-in-the-middle position you need to run the above configuration. This can be done by using the following commands on Kali Linux:
+To be able to get a Machine-in-the-Middle position you need to run the above configuration. This can be done by using the following commands on Kali Linux:
```

Comment:
```suggestion
To be able to get a MITM position you need to run the above configuration. This can be done by using the following commands on Kali Linux:
```